lib/tvdbapiv2/api_client.py

```diff
@@ -67,22 +67,26 @@ class ApiClient(object):
     :param header_name: a header to pass when making calls to the API.
     :param header_value: a header value to pass when making calls to the API.
     """
-    def __init__(self, host=None, header_name=None, header_value=None, cookie=None):
+    def __init__(self, host=None, cookie=None, session=None, user_agent=None, api_key=None):
 
         """
         Constructor of the class.
         """
-        self.rest_client = RESTClientObject()
+        self.session = session or requests.session()
+        if api_key:
+            self.session.auth = TVDBAuth(api_key)
```

Comment:
Some form of authentication is required for `TVDBAuth`, either a token or an apikey.